app.py

```diff
@@ -142,6 +142,25 @@ def apply_caching(response):
     return response
 
 
+# Static files
+# ===
+@app.route('/robots.txt')
+def robots():
+    return flask.send_from_directory(
+        app.static_folder, 'robots.txt')
+
+
+@app.route('/favicon.ico')
+def favicon():
+    size = flask.request.args.get('size', default=16, type=int)
```

Comment:
Given that `/favicon.ico` should *only* be called by browsers following the old standard (as they mostly still do), there's no use for `/favicon.ico?size=XX`. If we were deliberately asking for a specific favicon we should simply use the real URL - e.g. https://assets.ubuntu.com/v1/fdc99abe-ico_16px.png.

This means that in fact `/favicon.ico` should just redirect to one place.